project/src/main/webapp/gmailAuthLoad.js

```diff
@@ -0,0 +1,15 @@
+// const gmailAuth = new GmailAuthorization();
+function handleClientLoad() {
```

Comment:
It's better to only have one instance of GmailAuth, so can you make that a parameter for these functions and then you can use a global GmailAuth instance at the top of this file or in the window.onload function

Reply:
Ok new suggestion. Since `handleClientLoad` is called first and only once when the page loads, create a global instance of GmailAuth there.

```
let gmailAuth = null;

function handleClientLoad() {
   gmailAuth = new GmailAuthorization();
    gmailAuth.handleClientLoad();
}

function handleAuthClick() {
    gmailAuth.handleAuthClick();
}

function handleSignoutClick() {
    gmailAuth.handleSignoutClick();
} 
```

Reply:
it works!